tests/push/test_push.py

```diff
@@ -131,13 +131,13 @@ def test_typical_push(
     )
 
     # It should have invoked hook(s).
-    assert len(hookspy) == 24
+    assert len(hookspy) == 25
     (hook_name, hook_kwargs) = hookspy[0]
     assert hook_name == "task_start"
     (hook_name, hook_kwargs) = hookspy[1]
-    assert hook_name == "pulp_repository_pre_publish"
```

Comment:
Instead of changing the existing hook_name comparisons, could we please insert a new comparison?  i.e., before we tested hookspy 0 through 2, now let's test hookspy 0 through 3, so we still test everything we tested before but also the one new hook invocation.

Otherwise there is no testing of the invocation of pulp_repository_published hook (which is referenced in a comment on line 141, which would no longer make sense).